builders/interact/ethereum-api/wallets/talisman.md

```diff
@@ -0,0 +1,99 @@
+---
+title: How to Connect Talisman to Tanssi
+description: This guide walks you through how to connect Talisman, a comprehensive Polkadot, Substrate, and Ethereum wallet, to your Tanssi EVM ContainerChain. 
+---
+
+# Interacting with your Tanssi EVM ContainerChain Using Talisman
+
+## Introduction {: #introduction }
+
+Developers and users of Tanssi EVM ContainerChains have a variety of options when it comes to wallets. Thanks to their seamless Ethereum compatibility, Tanssi EVM ContainerChains support a great variety of popular wallets such as Talisman. 
+
+Talisman is a Web3 wallet that natively supports Substrate (Polkadot) and Ethereum accounts. This tutorial centers on the Ethereum API, but you can check out a similar [tutorial for interacting with Talisman using the Substrate API](/builders/interact/substrate-api/wallets/talisman){target=_blank}. The Talisman wallet browser extension is available on [Google Chrome](https://chrome.google.com/webstore/detail/talisman-polkadot-wallet/fijngjgcjhjmmpcmkeiomlglpeiijkld){target=_blank}, [Brave](https://chrome.google.com/webstore/detail/talisman-polkadot-wallet/fijngjgcjhjmmpcmkeiomlglpeiijkld){target=_blank}, and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/talisman-wallet-extension/){target=_blank}, and a corresponding asset dashboard is accessible at [app.talisman.xyz](https://app.talisman.xyz/){target=_blank}
+
+This guide takes you through all the necessary steps: from installing Talisman, to setting up a wallet, connecting it to your Tanssi EVM ContainerChain, and sending funds.
+
+## Setting up Talisman {: #setting-up-talisman }
+
+First, download and install the [Talisman extension](https://www.talisman.xyz/){target=_blank}. This guide will first cover creating a new wallet and later address importing an existing one. Review the terms and conditions then press **Get Started**. 
+
+![Get started with Talisman](/images/builders/interact/ethereum-api/wallets/talisman/talisman-1.png)
+
+On the following screen you'll be prompted to create a password to secure your new wallet. 
+
+![Enter password for Talisman Wallet](/images/builders/interact/ethereum-api/wallets/talisman/talisman-2.png)
+
+To create your first Ethereum account, take the following steps:
```

Comment:
This should be its own section "Creating an Ethereum Account" and then the importing section can be after this one